uiowa/uiowa
# Feature: People page bug fixes

# How to test

- `yarn workspace uids_base gulp --development`
- `blt ds --site=anthropology.uiowa.edu`
- `blt ds --site=journalism.uiowa.edu`
- Go to https://anthropology.local.drupal.uiowa.edu/people
- Verify that Form controls appear inline for safari at desktop breakpoint
- Verify that table row striping is gone
- Go to https://journalism.uiowa.edu/people and verify that page overflow is gone
